docs/CONTRIBUTING.md

```diff
@@ -46,13 +46,43 @@ make update-assets
 
 Commit and submit a PR.
 
-## Commit guidelines
+## Authoring PRs
 
-The title of the commit message should describe the _what_ about the
-changes. Additionally, it is helpful to add the _why_ in the body of
-the commit changes. Make sure to include any assumptions that you
-might have made in this commit. Changes to unrelated parts of the
-codebase should be kept as separate commits.
+These are general guidelines for making PRs/commits easier to review:
+
+ * Commits should be atomic and self-contained. Divide logically separate changes
```

Comment:
```suggestion
 * Commits should be atomic and self contained. Divide logically-separate changes
```

AFAIK hyphens in compound adjectives should be used when the adjective comes *after* the verb. Example:

:heavy_check_mark: This example is **self contained**.
:heavy_check_mark: I'm writing a **self-contained** example.

Reply:
Good to know! I always did what my gut feeling told me :see_no_evil: